dags/config/gcs_template_ingest_all_source.yaml

```diff
@@ -0,0 +1,43 @@
+source:
+  name: GCSTemplatedIngestAllSource
+  source_type: GCS
+  ingest_schedule: "@daily"
+  start_date: "2022-01-03"
+  catchup: False
+  acceptable_delay_minutes: 5
+  extra_options:
+    gcs_bucket: public-gcp-airflow-foundation-samples
+    file_source_config:
+      file_name_template: "{{ TABLE_NAME }}.csv"
+      source_format: CSV
+      delimeter: ","
+      file_prefix_filtering: False
+      delete_gcs_files: False
+      sensor_timeout: 6000
+  gcp_project: airflow-framework
+  location: us
+  dataset_data_name: af_test_ods
+  dataset_hds_override: af_test_hds
+  owner: test_user
+  notification_emails: []
+  landing_zone_options:
+    landing_zone_dataset: af_test_hds_landing_zone
+tables:
+  - table_name: users
+    ingestion_type: FULL
+    surrogate_keys: []
+    extra_options:
+      file_table_config:
+        directory_prefix: ""
+        allow_quoted_newlines: True
+templates:
+  - ingestion_type: FULL
+    surrogate_keys: {}
```

Comment:
can we make default be an empty list?

Reply:
why list? if we want the templates to have optional mappings for the surrogate keys, should be a dict, no?